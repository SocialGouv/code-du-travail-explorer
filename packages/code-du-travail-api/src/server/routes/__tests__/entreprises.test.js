test("fixme", () => {
  expect(true).toBeTruthy();
});

/*
const request = require("supertest");
const koa = require("koa");

const router = require("../entreprises");

const app = new koa();
const cb = app.callback();
app.use(router.routes());

const makePath = (query, address) =>
  `/api/v1/entreprises?q=${query}${address ? `&a=${address}` : ""}`;

const getFirst = (query, address) =>
  request(cb)
    .get(makePath(query, address))
    .then((resp) => resp.body.entreprises[0]);

const checkFields = ({
  address,
  activitePrincipale,
  conventions,
  simpleLabel,
  label,
  etablissements,
}) => {
  [
    address,
    etablissements,
    activitePrincipale,
    conventions,
    simpleLabel,
    label,
  ].forEach((f) => expect(f).toBeDefined());
};

test("search should return error 404 when no match", async () => {
  const response = await request(app.callback()).get(
    makePath("12345678901234567890")
  );
  expect(response.status).toBe(404);
});

test("search should return error 400 when no query", async () => {
  const response = await request(app.callback()).get(makePath(""));
  expect(response.status).toBe(400);
});

test("search no address", async () => {
  const response = await request(cb).get(makePath("michelin"));

  // check required fields are set accordingly
  response.body.entreprises.map(checkFields);

  expect(response.status).toBe(200);
  expect(response.body).toMatchSnapshot();
});

test("search case insensitive", async () => {
  const e1 = await getFirst("zara");
  const e2 = await getFirst("ZARA");

  expect(e1).toEqual(e1, e2);
});

test("search with postal code", async () => {
  const cp = "59800";
  const e = await getFirst("ZARA", cp);

  expect(e.address).toContain(cp);
  expect(e.matching).toBe(3);
});

test("search with city", async () => {
  const ville = "lille";
  const e = await getFirst("ZARA", ville);

  expect(e.address.toLowerCase()).toContain(ville);
});

test("search fuzzy", async () => {
  const e = await getFirst("mixhelin");

  expect(e.label).toContain("MICHELIN");
});

test("search highlight", async () => {
  const e = await getFirst("paul boulanger");

  expect(e.label).toContain("BOULANGERIES PAUL");
  expect(e.highlightLabel).toEqual(
    "<b><u>BOULANGERIES</b></u> <b><u>PAUL</b></u>"
  );
});

test("search SIREN / SIRET", async () => {
  const siret = "77560485300041";
  const siren = "775604853";

  const eSiret = await getFirst(siret);
  expect(eSiret.siret).toEqual(siret);
  expect(eSiret.siren).toEqual(siren);

  const eSiren = await getFirst(siren);
  expect(eSiren.siren).toEqual(siren);
});

test("conventions should be different if address set or not", async () => {
  const e1 = await getFirst("michelin");
  const e2 = await getFirst("michelin", "clermont");

  expect(e1.conventions.length).toEqual(2);
  expect(e2.conventions.length).toEqual(1);
});
*/
