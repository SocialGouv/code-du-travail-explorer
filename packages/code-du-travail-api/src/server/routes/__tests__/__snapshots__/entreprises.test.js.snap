// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search no address 1`] = `
Object {
  "entreprises": Array [
    Object {
      "activitePrincipale": "Analyses, essais et inspections techniques",
      "address": "Route d'Arles 13300 Salon-de-Provence",
      "convention": "Caoutchouc",
      "etablissements": "28",
      "highlightLabel": "MANUF FRANC PNEUMATIQ <b><u>MICHELIN</b></u>",
      "id": "855200507",
      "idcc": 45,
      "label": "MANUF FRANC PNEUMATIQ MICHELIN",
      "simpleLabel": "MANUF FRANC PNEUMATIQ MICHELIN",
      "siren": "855200507",
      "siret": "85520050703094",
    },
    Object {
      "activitePrincipale": "Fabrication de caoutchouc synthétique",
      "address": "Rue Edouard Michelin 33530 Bassens",
      "convention": "Caoutchouc",
      "etablissements": "2",
      "highlightLabel": "SIMOREP ET CIE SOC DU CAOUTCHOUC SYNTHETIQUE <b><u>MICHELIN</b></u>",
      "id": "328525746",
      "idcc": 45,
      "label": "SIMOREP ET CIE SOC DU CAOUTCHOUC SYNTHETIQUE MICHELIN",
      "simpleLabel": "SIMOREP ET CIE",
      "siren": "328525746",
      "siret": "32852574600017",
    },
    Object {
      "activitePrincipale": "Programmation informatique",
      "address": "27 Cours de l'Ile Seguin 92100 Boulogne-Billancourt",
      "convention": "Caoutchouc",
      "etablissements": "2",
      "highlightLabel": "<b><u>MICHELIN</b></u> TRAVEL PARTNER",
      "id": "433677721",
      "idcc": 45,
      "label": "MICHELIN TRAVEL PARTNER",
      "simpleLabel": "MICHELIN TRAVEL PARTNER",
      "siren": "433677721",
      "siret": "43367772100030",
    },
    Object {
      "activitePrincipale": "Aide à domicile",
      "address": "Allee de Marveille 09350 Les Bordes-sur-Arize",
      "convention": "Hospitalisation privée : établissements privés d'hospitalisation, de soins, de cure et de garde à but non lucratif (FEHAP)",
      "etablissements": "6",
      "highlightLabel": "S.S.I.D.P.A <b><u>MICHELINE</b></u> GOYHENECHE SOINS DOMICILE PERSONNES AGEES SERVICE AIDE",
      "id": "333800407",
      "idcc": 29,
      "label": "S.S.I.D.P.A MICHELINE GOYHENECHE SOINS DOMICILE PERSONNES AGEES SERVICE AIDE",
      "simpleLabel": "MICHELINE GOYHENECHE",
      "siren": "333800407",
      "siret": "33380040700053",
    },
    Object {
      "activitePrincipale": "Activités des sièges sociaux",
      "address": "63 Rue Henri Barbusse 63000 Clermont-Ferrand",
      "convention": "Hospitalisation privée : établissements privés d'hospitalisation, de soins, de cure et de garde à but non lucratif (FEHAP)",
      "etablissements": "3",
      "highlightLabel": "SOHPEM SOC OEUVRES HYGIENE <b><u>MICHELIN</b></u>",
      "id": "857201727",
      "idcc": 29,
      "label": "SOHPEM SOC OEUVRES HYGIENE MICHELIN",
      "simpleLabel": "SOC OEUVRES HYGIENE MICHELIN",
      "siren": "857201727",
      "siret": "85720172700180",
    },
    Object {
      "activitePrincipale": "Autres assurances",
      "address": "2 Rue de Bien - Assis 63100 Clermont-Ferrand",
      "convention": "Mutualité",
      "etablissements": "1",
      "highlightLabel": "MNPEM MUTUELLE NATIONALE PERSO ETS <b><u>MICHELIN</b></u>",
      "id": "382046464",
      "idcc": 2128,
      "label": "MNPEM MUTUELLE NATIONALE PERSO ETS MICHELIN",
      "simpleLabel": "MUTUELLE NATIONALE PERSO ETS MICHELIN",
      "siren": "382046464",
      "siret": "38204646400030",
    },
    Object {
      "activitePrincipale": "Transports aériens de passagers",
      "address": "23 Place des Carmes Dechaux 63000 Clermont-Ferrand",
      "convention": "Caoutchouc",
      "etablissements": "3",
      "highlightLabel": "<b><u>MICHELIN</b></u> AIR SERVICES",
      "id": "414746503",
      "idcc": 45,
      "label": "MICHELIN AIR SERVICES",
      "simpleLabel": "MICHELIN AIR SERVICES",
      "siren": "414746503",
      "siret": "41474650300013",
    },
    Object {
      "activitePrincipale": "Activités des syndicats de salariés",
      "address": "16 Rue de Toutlemonde 49300 Cholet",
      "convention": "Caoutchouc",
      "etablissements": "7",
      "highlightLabel": "COMITE ENTREPRISE <b><u>MICHELIN</b></u>",
      "id": "775634207",
      "idcc": 45,
      "label": "COMITE ENTREPRISE MICHELIN",
      "simpleLabel": "COMITE ENTREPRISE MICHELIN",
      "siren": "775634207",
      "siret": "77563420700093",
    },
    Object {
      "activitePrincipale": "Activités des syndicats de salariés",
      "address": "6 Avenue du Président René Coty 10600 La Chapelle-Saint-Luc",
      "convention": "Caoutchouc",
      "etablissements": "1",
      "highlightLabel": "COMITE D'ETABLISSEMENT <b><u>MICHELIN</b></u>",
      "id": "327798021",
      "idcc": 45,
      "label": "COMITE D'ETABLISSEMENT MICHELIN",
      "simpleLabel": "COMITE D'ETABLISSEMENT MICHELIN",
      "siren": "327798021",
      "siret": "32779802100017",
    },
    Object {
      "activitePrincipale": "Autres commerces de détail alimentaires en magasin spécialisé",
      "address": "15 Grand Rue 01210 Ferney-Voltaire",
      "convention": "Commerce de détail des fruits et légumes, épicerie et produits laitiers",
      "etablissements": "3",
      "highlightLabel": "FROMAGERIE DANIEL <b><u>MICHELIN</b></u>",
      "id": "448695676",
      "idcc": 1505,
      "label": "FROMAGERIE DANIEL MICHELIN",
      "simpleLabel": "FROMAGERIE DANIEL MICHELIN",
      "siren": "448695676",
      "siret": "44869567600032",
    },
    Object {
      "activitePrincipale": "Commerce de détail de viandes et de produits à base de viande en magasin spécialisé",
      "address": "68 Rue Louis le Grand 39140 Bletterans",
      "convention": "Boucherie, boucherie-charcuterie, boucherie hippophagique, triperie, commerces de volailles et gibiers",
      "etablissements": "2",
      "highlightLabel": "SARL ROUFFIAC - <b><u>MICHELIN</b></u>",
      "id": "377672563",
      "idcc": 992,
      "label": "SARL ROUFFIAC - MICHELIN",
      "simpleLabel": "SARL ROUFFIAC - MICHELIN",
      "siren": "377672563",
      "siret": "37767256300014",
    },
    Object {
      "activitePrincipale": "Fabrication de fromage",
      "address": "2 Place de la Mairie 25160 Saint-Point-Lac",
      "convention": "Lait : industrie laitière",
      "etablissements": "2",
      "highlightLabel": "FROMAGERIE <b><u>MICHELIN</b></u> FABRICE",
      "id": "478481039",
      "idcc": 112,
      "label": "FROMAGERIE MICHELIN FABRICE",
      "simpleLabel": "FROMAGERIE MICHELIN FABRICE",
      "siren": "478481039",
      "siret": "47848103900012",
    },
    Object {
      "activitePrincipale": "Travaux de plâtrerie",
      "address": "2 Avenue de Chalon 71380 Saint-Marcel",
      "etablissements": "1",
      "highlightLabel": "PLATRERIE PEINTURE <b><u>MICHELIN</b></u>",
      "id": "518948674",
      "idcc": 152,
      "label": "PLATRERIE PEINTURE MICHELIN",
      "simpleLabel": "PLATRERIE PEINTURE MICHELIN",
      "siren": "518948674",
      "siret": "51894867400040",
    },
    Object {
      "activitePrincipale": "Travaux de maçonnerie générale et gros œuvre de bâtiment",
      "address": "Chemin de la Batterie 42220 Graix",
      "convention": "Bâtiment Ouvriers (entreprises occupant jusqu'à 10 salariés)",
      "etablissements": "1",
      "highlightLabel": "DAVID ET HERVE <b><u>MICHELIN</b></u>",
      "id": "404299877",
      "idcc": 1596,
      "label": "DAVID ET HERVE MICHELIN",
      "simpleLabel": "DAVID ET HERVE MICHELIN",
      "siren": "404299877",
      "siret": "40429987700013",
    },
    Object {
      "activitePrincipale": "Élevage d’autres bovins et de buffles",
      "address": "10 bis Rue de Champagne 51330 Vroil",
      "etablissements": "2",
      "highlightLabel": "<b><u>MICHELIN</b></u>",
      "id": "390253904",
      "idcc": 8214,
      "label": "MICHELIN",
      "simpleLabel": "MICHELIN",
      "siren": "390253904",
      "siret": "39025390400010",
    },
    Object {
      "activitePrincipale": "Activités de soutien aux cultures",
      "address": "1 Rue du Champ Lain 21190 Tailly",
      "etablissements": "1",
      "highlightLabel": "<b><u>MICHELIN</b></u> JOEL",
      "id": "384460986",
      "idcc": 8262,
      "label": "MICHELIN JOEL",
      "simpleLabel": "MICHELIN JOEL",
      "siren": "384460986",
      "siret": "38446098600027",
    },
    Object {
      "activitePrincipale": "Activités d’architecture",
      "address": "Rue du Carreau 69960 Corbas",
      "convention": "Bâtiment Ouvriers (entreprises occupant jusqu'à 10 salariés)",
      "etablissements": "1",
      "highlightLabel": "PARM CONCEPT PHARMACIE REALISATION <b><u>MICHELIN</b></u>",
      "id": "509533600",
      "idcc": 1596,
      "label": "PARM CONCEPT PHARMACIE REALISATION MICHELIN",
      "simpleLabel": "PARM CONCEPT",
      "siren": "509533600",
      "siret": "50953360000011",
    },
    Object {
      "activitePrincipale": "Activités des sièges sociaux",
      "address": "23 Place des Carmes Dechaux 63000 Clermont-Ferrand",
      "convention": "Caoutchouc",
      "etablissements": "2",
      "highlightLabel": "<b><u>MICHELIN</b></u> TYRE RESEARCH",
      "id": "834791360",
      "idcc": 45,
      "label": "MICHELIN TYRE RESEARCH",
      "simpleLabel": "MICHELIN TYRE RESEARCH",
      "siren": "834791360",
      "siret": "83479136000029",
    },
    Object {
      "activitePrincipale": "Activités des sièges sociaux",
      "address": "Place des Carmes Dechaux 63000 Clermont-Ferrand",
      "convention": "Caoutchouc",
      "etablissements": "2",
      "highlightLabel": "<b><u>MICHELIN</b></u> INDIA TECHNOLOGY CENTER LLP",
      "id": "834830069",
      "idcc": 45,
      "label": "MICHELIN INDIA TECHNOLOGY CENTER LLP",
      "simpleLabel": "MICHELIN INDIA TECHNOLOGY CENTER LLP",
      "siren": "834830069",
      "siret": "83483006900029",
    },
    Object {
      "activitePrincipale": "Restauration traditionnelle",
      "address": "10 Rue du Louvre 75001 Paris",
      "convention": "Restauration rapide",
      "etablissements": "1",
      "highlightLabel": "CHEZ <b><u>MICHELINE</b></u>",
      "id": "823227624",
      "idcc": 1501,
      "label": "CHEZ MICHELINE",
      "simpleLabel": "CHEZ MICHELINE",
      "siren": "823227624",
      "siret": "82322762400017",
    },
    Object {
      "activitePrincipale": "Commerce de voitures et de véhicules automobiles légers",
      "address": "21 Rue du 8 Mai 72220 Teloché",
      "convention": "Services de l'automobile (Commerce et réparation de l'automobile, du cycle et du motocycle, activités connexes, contrôle technique automobile, formation des conducteurs)",
      "etablissements": "1",
      "highlightLabel": "GARAGE <b><u>MICHELIN</b></u> ANTHONY",
      "id": "393030010",
      "idcc": 1090,
      "label": "GARAGE MICHELIN ANTHONY",
      "simpleLabel": "GARAGE MICHELIN ANTHONY",
      "siren": "393030010",
      "siret": "39303001000028",
    },
    Object {
      "activitePrincipale": "Coiffure",
      "address": "74 Rue Carnot 89500 Villeneuve-sur-Yonne",
      "convention": "Coiffure et professions connexes",
      "etablissements": "1",
      "highlightLabel": "BOULEAU FLE <b><u>MICHELINE</b></u> COIFFURE",
      "id": "325280840",
      "idcc": 2596,
      "label": "BOULEAU FLE MICHELINE COIFFURE",
      "simpleLabel": "BOULEAU",
      "siren": "325280840",
      "siret": "32528084000019",
    },
    Object {
      "activitePrincipale": "Activités des syndicats de salariés",
      "address": "Avenue Edouard Michelin 56000 Vannes",
      "convention": "Caoutchouc",
      "etablissements": "1",
      "highlightLabel": "COMITE D ETABL <b><u>MICHELIN</b></u>",
      "id": "330211707",
      "idcc": 45,
      "label": "COMITE D ETABL MICHELIN",
      "simpleLabel": "COMITE D ETABL MICHELIN",
      "siren": "330211707",
      "siret": "33021170700014",
    },
    Object {
      "activitePrincipale": "Activités des professionnels de la rééducation, de l’appareillage et des pédicures-podologues",
      "address": "6 Rue Charles Arnoult 21700 Nuits-Saint-Georges",
      "convention": "Personnel des cabinets médicaux",
      "etablissements": "1",
      "highlightLabel": "<b><u>MICHELIN</b></u>",
      "id": "352083158",
      "idcc": 1147,
      "label": "MICHELIN",
      "simpleLabel": "MICHELIN",
      "siren": "352083158",
      "siret": "35208315800068",
    },
    Object {
      "activitePrincipale": "Activités des agents et courtiers d’assurances",
      "address": "33 Cours de la République 33390 Blaye",
      "convention": "Personnels des agences générales d'assurances",
      "etablissements": "1",
      "highlightLabel": "<b><u>MICHELIN</b></u>",
      "id": "390475879",
      "idcc": 2335,
      "label": "MICHELIN",
      "simpleLabel": "MICHELIN",
      "siren": "390475879",
      "siret": "39047587900016",
    },
    Object {
      "activitePrincipale": "Commerce de détail de produits pharmaceutiques en magasin spécialisé",
      "address": "",
      "convention": "Pharmacie d'officine",
      "etablissements": "1",
      "highlightLabel": "RIGARD-CERISON <b><u>MICHELIN</b></u> PHARMACIE",
      "id": "342339397",
      "idcc": 1996,
      "label": "RIGARD-CERISON MICHELIN PHARMACIE",
      "simpleLabel": "RIGARD-CERISON",
      "siren": "342339397",
      "siret": "34233939700032",
    },
    Object {
      "activitePrincipale": "Activités des agents et courtiers d’assurances",
      "address": "241 Rue de Tolbiac 75013 Paris",
      "convention": "Entreprises de courtage d'assurances et/ou de réassurances",
      "etablissements": "1",
      "highlightLabel": "AMCF <b><u>MICHELINE</b></u> AKNIN CONSEIL FIDELISATION",
      "id": "344208301",
      "idcc": 2247,
      "label": "AMCF MICHELINE AKNIN CONSEIL FIDELISATION",
      "simpleLabel": "MICHELINE AKNIN CONSEIL FIDELISATION",
      "siren": "344208301",
      "siret": "34420830100012",
    },
    Object {
      "activitePrincipale": "Restauration traditionnelle",
      "address": "6 Rue Saint-martin (La Baroche-sous-Lucé) 61330 Juvigny Val d'Andaine",
      "convention": "Hôtels, cafés, restaurants",
      "etablissements": "2",
      "highlightLabel": "LEBATARD CHEZ MIMI LEROUX <b><u>MICHELINE</b></u>",
      "id": "345080444",
      "idcc": 1979,
      "label": "LEBATARD CHEZ MIMI LEROUX MICHELINE",
      "simpleLabel": "LEBATARD",
      "siren": "345080444",
      "siret": "34508044400036",
    },
    Object {
      "activitePrincipale": "Fabrication d’autres outillages",
      "address": "Route d'Allerey 71590 Gergy",
      "convention": "Convention collective départementale des industries de la métallurgie de Saône-et-Loire",
      "etablissements": "1",
      "highlightLabel": "RMGI RENIAUX <b><u>MICHELIN</b></u> GRAVURES INDUSTRIELLES",
      "id": "378896062",
      "idcc": 1564,
      "label": "RMGI RENIAUX MICHELIN GRAVURES INDUSTRIELLES",
      "simpleLabel": "RENIAUX MICHELIN GRAVURES INDUSTRIELLES",
      "siren": "378896062",
      "siret": "37889606200015",
    },
    Object {
      "activitePrincipale": "Activités combinées de soutien lié aux bâtiments",
      "address": "24 Square de Clignancourt 75018 Paris",
      "convention": "Gardiens, concierges et employés d'immeubles",
      "etablissements": "1",
      "highlightLabel": "<b><u>MICHELINE</b></u> STERVINOU",
      "id": "39020748",
      "idcc": 1043,
      "label": "MICHELINE STERVINOU",
      "simpleLabel": "MICHELINE STERVINOU",
      "siren": "39020748",
      "siret": "3902074800016",
    },
    Object {
      "activitePrincipale": "Activités combinées de soutien lié aux bâtiments",
      "address": "52 Rue de Ponthieu 75008 Paris",
      "convention": "Gardiens, concierges et employés d'immeubles",
      "etablissements": "1",
      "highlightLabel": "MME JARLAND DENISE  WANGER <b><u>MICHELIN</b></u>",
      "id": "39031745",
      "idcc": 1043,
      "label": "MME JARLAND DENISE  WANGER MICHELIN",
      "simpleLabel": "MME JARLAND DENISE  MME WANGER MICHELIN",
      "siren": "39031745",
      "siret": "3903174500019",
    },
    Object {
      "activitePrincipale": "Commerce de détail de viandes et de produits à base de viande en magasin spécialisé",
      "address": "67 Rue Ignace Humblot 62138 Auchy-les-Mines",
      "convention": "Boucherie, boucherie-charcuterie, boucherie hippophagique, triperie, commerces de volailles et gibiers",
      "etablissements": "1",
      "highlightLabel": "BOUCHERIE CHARCUTERIE YOANN ET <b><u>MICHELINE</b></u>",
      "id": "428912901",
      "idcc": 992,
      "label": "BOUCHERIE CHARCUTERIE YOANN ET MICHELINE",
      "simpleLabel": "BOUCHERIE CHARCUTERIE YOANN ET MICHELINE",
      "siren": "428912901",
      "siret": "42891290100025",
    },
    Object {
      "activitePrincipale": "Activités combinées de soutien lié aux bâtiments",
      "address": "11 Rue des Deux Ponts 75004 Paris",
      "convention": "Gardiens, concierges et employés d'immeubles",
      "etablissements": "1",
      "highlightLabel": "SYNDICAT COPRO KRIEGEL <b><u>MICHELINE</b></u>",
      "id": "43899103",
      "idcc": 1043,
      "label": "SYNDICAT COPRO KRIEGEL MICHELINE",
      "simpleLabel": "SYNDICAT COPRO KRIEGEL MICHELINE",
      "siren": "43899103",
      "siret": "4389910300014",
    },
    Object {
      "activitePrincipale": "Activités combinées de soutien lié aux bâtiments",
      "address": "15 Rue Cino Del Duca 75017 Paris",
      "convention": "Gardiens, concierges et employés d'immeubles",
      "etablissements": "1",
      "highlightLabel": "<b><u>MICHELINE</b></u> PITTARD",
      "id": "39289897",
      "idcc": 1043,
      "label": "MICHELINE PITTARD",
      "simpleLabel": "MICHELINE PITTARD",
      "siren": "39289897",
      "siret": "3928989700017",
    },
    Object {
      "activitePrincipale": "Activités combinées de soutien lié aux bâtiments",
      "address": "19 Avenue Léopold II 75016 Paris",
      "convention": "Gardiens, concierges et employés d'immeubles",
      "etablissements": "1",
      "highlightLabel": "<b><u>MICHELINE</b></u> REBUT",
      "id": "39182951",
      "idcc": 1043,
      "label": "MICHELINE REBUT",
      "simpleLabel": "MICHELINE REBUT",
      "siren": "39182951",
      "siret": "3918295100010",
    },
    Object {
      "activitePrincipale": "Activités des agences de voyage",
      "address": "282 Rue Nationale 36400 La Châtre",
      "convention": "Agences de voyages et de tourisme : personnels, guides accompagnateurs et accompagnateurs",
      "etablissements": "1",
      "highlightLabel": "LES VOYAGES DE <b><u>MICHELINE</b></u>",
      "id": "793598350",
      "idcc": 1710,
      "label": "LES VOYAGES DE MICHELINE",
      "simpleLabel": "LES VOYAGES DE MICHELINE",
      "siren": "793598350",
      "siret": "79359835000015",
    },
    Object {
      "activitePrincipale": "Restauration traditionnelle",
      "address": "14 Place Carnot 69002 Lyon",
      "convention": "Hôtels, cafés, restaurants",
      "etablissements": "1",
      "highlightLabel": "L'INTERLUDE CHEZ <b><u>MICHELINE</b></u>",
      "id": "514193903",
      "idcc": 1979,
      "label": "L'INTERLUDE CHEZ MICHELINE",
      "simpleLabel": "L'INTERLUDE",
      "siren": "514193903",
      "siret": "51419390300028",
    },
    Object {
      "activitePrincipale": "Commerce de détail de textiles en magasin spécialisé",
      "address": "19 Rue du Palais 57000 Metz",
      "convention": "Commerce de détail de l'habillement et des articles textiles",
      "etablissements": "1",
      "highlightLabel": "<b><u>MICHELIN</b></u> JEANPIERRE L'EPINGLE",
      "id": "488342593",
      "idcc": 1483,
      "label": "MICHELIN JEANPIERRE L'EPINGLE",
      "simpleLabel": "MICHELIN",
      "siren": "488342593",
      "siret": "48834259300016",
    },
    Object {
      "activitePrincipale": "Agences immobilières",
      "address": "2 Rue Maréchal Foch 21120 Is-sur-Tille",
      "convention": "Immobilier : administrateurs de biens, sociétés immobilières, agents immobiliers",
      "etablissements": "1",
      "highlightLabel": "<b><u>MICHELIN</b></u> DIJONNORD-IMMO.FR AGENCE D'IS-SUR-T",
      "id": "433361383",
      "idcc": 1527,
      "label": "MICHELIN DIJONNORD-IMMO.FR AGENCE D'IS-SUR-T",
      "simpleLabel": "DIJONNORD-IMMO.FR",
      "siren": "433361383",
      "siret": "43336138300030",
    },
    Object {
      "activitePrincipale": "Restauration traditionnelle",
      "address": "4 Avenue Grimeler 77330 Ozoir-la-Ferrière",
      "convention": "Hôtels, cafés, restaurants",
      "etablissements": "1",
      "highlightLabel": "CHEZ <b><u>MICHELINE</b></u> ET PAULETTE",
      "id": "813772530",
      "idcc": 1979,
      "label": "CHEZ MICHELINE ET PAULETTE",
      "simpleLabel": "CHEZ MICHELINE ET PAULETTE",
      "siren": "813772530",
      "siret": "81377253000016",
    },
    Object {
      "activitePrincipale": "Autres organisations fonctionnant par adhésion volontaire",
      "address": "116 Rue de la Tour 75016 Paris",
      "convention": "Caoutchouc",
      "etablissements": "2",
      "highlightLabel": "FONDATION D'ENTREPRISE <b><u>MICHELIN</b></u>",
      "id": "804046159",
      "idcc": 45,
      "label": "FONDATION D'ENTREPRISE MICHELIN",
      "simpleLabel": "FONDATION D'ENTREPRISE MICHELIN",
      "siren": "804046159",
      "siret": "80404615900021",
    },
    Object {
      "activitePrincipale": "Activités des sièges sociaux",
      "address": "23 Place des Carmes Dechaux 63000 Clermont-Ferrand",
      "convention": "Caoutchouc",
      "etablissements": "2",
      "highlightLabel": "<b><u>MICHELIN</b></u> SIAM COMPANY LTD",
      "id": "834830044",
      "idcc": 45,
      "label": "MICHELIN SIAM COMPANY LTD",
      "simpleLabel": "MICHELIN SIAM COMPANY LTD",
      "siren": "834830044",
      "siret": "83483004400022",
    },
    Object {
      "activitePrincipale": "Activités des sièges sociaux",
      "address": "23 Place des Carmes Dechaux 63000 Clermont-Ferrand",
      "convention": "Caoutchouc",
      "etablissements": "2",
      "highlightLabel": "<b><u>MICHELIN</b></u> INDIA PRIVATE LTD",
      "id": "834821480",
      "idcc": 45,
      "label": "MICHELIN INDIA PRIVATE LTD",
      "simpleLabel": "MICHELIN INDIA PRIVATE LTD",
      "siren": "834821480",
      "siret": "83482148000029",
    },
    Object {
      "activitePrincipale": "Activités d’architecture",
      "address": "47 Rue Ampère 75017 Paris",
      "convention": "Entreprises d'architecture",
      "etablissements": "1",
      "highlightLabel": "<b><u>MICHELIN</b></u>",
      "id": "324726306",
      "idcc": 2332,
      "label": "MICHELIN",
      "simpleLabel": "MICHELIN",
      "siren": "324726306",
      "siret": "32472630600016",
    },
    Object {
      "activitePrincipale": "Activités combinées de soutien lié aux bâtiments",
      "address": "28 Rue du Faubourg-de-saverne 67000 Strasbourg",
      "convention": "Gardiens, concierges et employés d'immeubles",
      "etablissements": "1",
      "highlightLabel": "SYNDIC COPROP 28 R DU FG DE SAVERNE MME LAVALLEE <b><u>MICHELINE</b></u>",
      "id": "352034706",
      "idcc": 1043,
      "label": "SYNDIC COPROP 28 R DU FG DE SAVERNE MME LAVALLEE MICHELINE",
      "simpleLabel": "SYNDIC COPROP 28 R DU FG DE SAVERNE",
      "siren": "352034706",
      "siret": "35203470600015",
    },
    Object {
      "activitePrincipale": "Supports juridiques de gestion de patrimoine mobilier",
      "address": "Le Clus Ouest 74540 Gruffy",
      "convention": "Gardiens, concierges et employés d'immeubles",
      "etablissements": "1",
      "highlightLabel": "<b><u>MICHELIN</b></u> SAFFRAY",
      "id": "443174073",
      "idcc": 1043,
      "label": "MICHELIN SAFFRAY",
      "simpleLabel": "MICHELIN SAFFRAY",
      "siren": "443174073",
      "siret": "44317407300017",
    },
    Object {
      "activitePrincipale": "Production de films institutionnels et publicitaires",
      "address": "188 Boulevard Pereire 75017 Paris",
      "convention": "Production audiovisuelle",
      "etablissements": "1",
      "highlightLabel": "LES <b><u>MICHELINES</b></u>",
      "id": "848672226",
      "idcc": 2642,
      "label": "LES MICHELINES",
      "simpleLabel": "LES MICHELINES",
      "siren": "848672226",
      "siret": "84867222600018",
    },
    Object {
      "activitePrincipale": "Activités des sièges sociaux",
      "address": "23 Place des Carmes Dechaux 63000 Clermont-Ferrand",
      "convention": "Caoutchouc",
      "etablissements": "2",
      "highlightLabel": "<b><u>MICHELIN</b></u> ROH CO LTD",
      "id": "834790800",
      "idcc": 45,
      "label": "MICHELIN ROH CO LTD",
      "simpleLabel": "MICHELIN ROH CO LTD",
      "siren": "834790800",
      "siret": "83479080000025",
    },
    Object {
      "activitePrincipale": "Activités des sièges sociaux",
      "address": "Place des Carmes Dechaux 63000 Clermont-Ferrand",
      "convention": "Caoutchouc",
      "etablissements": "2",
      "highlightLabel": "<b><u>MICHELIN</b></u> CHINA INVESTMENT COMPANY LTD",
      "id": "834790834",
      "idcc": 45,
      "label": "MICHELIN CHINA INVESTMENT COMPANY LTD",
      "simpleLabel": "MICHELIN CHINA INVESTMENT COMPANY LTD",
      "siren": "834790834",
      "siret": "83479083400024",
    },
    Object {
      "activitePrincipale": "Activités des sièges sociaux",
      "address": "23 Place des Carmes Dechaux 63000 Clermont-Ferrand",
      "convention": "Caoutchouc",
      "etablissements": "2",
      "highlightLabel": "<b><u>MICHELIN</b></u> AIM FZE",
      "id": "834680969",
      "idcc": 45,
      "label": "MICHELIN AIM FZE",
      "simpleLabel": "MICHELIN AIM FZE",
      "siren": "834680969",
      "siret": "83468096900021",
    },
  ],
}
`;
